src/ProtectedRoomsSet.ts

```diff
@@ -92,7 +92,12 @@ export class ProtectedRoomsSet {
      * Intended to be `this.syncWithUpdatedPolicyList` so we can add it in `this.watchList` and remove it in `this.unwatchList`.
      * Otherwise we would risk being informed about lists we no longer watch.
      */
-    private readonly listUpdateListener: (list: PolicyList, changes: ListRuleChange[]) => void;
+    private readonly listUpdateListener: (list: PolicyList, changes: ListRuleChange[], revisionId: string) => void;
+
+    /**
+     * The revision of a each watched list that we have applied to protected rooms.
```

Comment:
Is that a revision id? If so, could you clarify this in the doc?

For bonus points, you could make it a newtype :)